resolwe_api/sample.py

```diff
@@ -0,0 +1,138 @@
+"""Sample"""
```

Comment:
Maybe we should have a module called `resources` that would contain `data.py`, `sample.py`, `collection.py`, and later `study.py`.